fern/docs/pages/user-guide/etcd-metadata-exchange.md

```
@@ -0,0 +1,320 @@
+nixlAgent agent("my_agent", cfg);
+```
+
+```python title="Python"
+config = nixl_agent_config(
+    enable_prog_thread=True,
+    enable_listen_thread=True
+)
+# etcdWatchTimeout is configured at the C++ level;
+# Python uses the default (5 seconds) unless set via the C++ bindings
+agent = nixl_agent("my_agent", config)
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)